okhttp/src/test/java/okhttp3/CacheTest.java

```diff
@@ -1936,8 +1936,7 @@ private RecordedRequest assertClientSuppliedCondition(MockResponse seed, String
         (double) 0, offset(250.0));
 
     // A full cache hit reads the cache.
-    Thread.sleep(500); // Make sure t1 and t2 are distinct.
-    long t2 = System.currentTimeMillis();
+    Thread.sleep(500);
```

Comment:
Can you drop this to 10?  And rename originalTimestamp to updateTimestamp.

Reply:
done! :)